UPDATING.md

```diff
@@ -154,6 +153,480 @@ The Old and New provider configuration keys that have changed are as follows
 
 For more information, visit https://flask-appbuilder.readthedocs.io/en/latest/security.html#authentication-oauth
 
+### Changes to the KubernetesExecutor
+
+#### The KubernetesExecutor Will No Longer Read from the airflow.cfg for Base Pod Configurations
```

Comment:
Can you add a full list of configuration options that have been deleted? This is important if we are going to work on `airflow upgrade-check`, because having the list we can detect incorrect configuration and notify the user at runtime.

Reply:
@mik-laj added